Running-Mastodon/Production-guide.md

```diff
@@ -37,7 +37,9 @@ server {
   server_name example.com;
 
   ssl_protocols TLSv1.2;
-  ssl_ciphers HIGH:!MEDIUM:!LOW:!aNULL:!NULL:!SHA;
+  ssl_ciphers 'ECDHE+CHACHA20:ECDHE+AESGCM'; # or 'ECDHE:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL'
+  #ssl_dhparam ssl/dhparam.pem; # generate with “openssl dhparam -out dhparam.pem 4096”, only needed if you add DHE to the ciphers
+  ssl_ecdh_curve X25519:sect571r1:secp521r1:secp384r1:prime256v1; # If only one curve : prime256v1 (Android 7.0 compat) or secp384r1 (not compatible with Android 7.0)
```

Comment:
The best choice is probably to put nothing, then, because as for now Debian and Ubuntu don't have Nginx 1.11+ until you install it from a third-party, and OpenSSL 1.0.2 is not on Debian 8 and Ubuntu <= 14.04